mkosi/config.py

```diff
@@ -3869,29 +3881,31 @@ def parse_config_one(self, path: Path, profiles: bool = False, local: bool = Fal
                 setattr(self.config, s.dest, s.parse(v, getattr(self.config, s.dest, None)))
                 self.parse_new_includes()
 
+        profilepath = None
         if profiles:
             profile = self.finalize_value(SETTINGS_LOOKUP_BY_DEST["profile"])
             self.immutable.add("Profile")
 
             if profile:
-                for p in (profile, f"{profile}.conf"):
-                    p = Path("mkosi.profiles") / p
-                    if p.exists():
+                for p in (Path(profile), Path(f"{profile}.conf")):
+                    profilepath = Path("mkosi.profiles") / p
+                    if profilepath.exists():
                         break
                 else:
                     die(f"Profile '{profile}' not found in mkosi.profiles/")
```

Comment:
```suggestion
                if not any((Path("mkosi.profiles") / p).exists() for p in (Path(profile), Path(f"{profile}.conf")))
                    die(f"Profile '{profile}' not found in mkosi.profiles/")
```

Reply:
Does this work? We still need the profile path later.